app/views/school/la_funded_places/show.html.erb

```diff
@@ -0,0 +1,76 @@
+<%- title = t('page_titles.iss.order_devices') %>
+<%- content_for :title, title %>
+<%- content_for :before_content do %>
+  <%- school_breadcrumbs items: title, school: @school, user: impersonated_or_current_user %>
+<%- end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= title %></h1>
+
+    <%= render Computacenter::TechSourceMaintenanceBannerComponent.new %>
+    <%= render partial: 'shared/half_term_delivery_suspension' %>
+
+    <%= render DeviceCountComponent.new(school: @school) %>
+
+    <p class="govuk-body">
+      You’ve been allocated <%= "#{@school.std_device_allocation.cap} #{'laptop'.pluralize(@school.std_device_allocation.cap)}" %>.
+    </p>
+    <%= govuk_details summary: 'How your allocation has been calculated' do %>
+      <p class="govuk-body">
+        Your allocation estimate is based on the data <%= @school.responsible_body.name %>
+        submitted in the latest published alternative provision census (2020).
+      </p>
+      <p class="govuk-body">
+        We’ve allocated one laptop for each pupil in years 3 to 11 that
+        <%= @school.responsible_body.name %> reported as having a state-funded
+        place at an independent setting, and also being eligible for free school meals.
```

Comment:
... at an independent special school or alternative provision, and also ..

Reply:
Done